src/components/ConnectedFilter/utils.js

```diff
@@ -8,7 +8,7 @@ export const getFilterGroupConfig = (filterConfig) => ({
   })),
 });
 
-const getSingleFilterOption = (histogramResult, initHistogramRes) => {
+const getSingleFilterOption = (histogramResult, initHistogramRes, guppyConfig) => {
```

Comment:
you can just put `explorerHideDataFilter` in here rather than passing in the entire `guppyConfig` object